Pilot Data Exploration.R

```diff
@@ -24,46 +24,213 @@ if(pull_fresh_dataset) {
+t.test(b_pi_completers$b_shs_agency_sum, b_pi_completers$pi_shs_agency_sum, paired = TRUE)
+b_pi_completers %>%
+  select(b_shs_agency_sum, pi_shs_agency_sum) %>%
+  mutate(shs_agency_sum_change = pi_shs_agency_sum - b_shs_agency_sum) %>%
+  summarize(across(everything(), list(mean = mean, sd = sd))) %>%
+  mutate(d_z = shs_agency_sum_change_mean / shs_agency_sum_change_sd,
+         d_av = shs_agency_sum_change_mean / ((b_shs_agency_sum_sd + pi_shs_agency_sum_sd) / 2))
+
+## Functionality Appreciation Scale
+plot(b_pi_completers$b_fas_sum, b_pi_completers$pi_fas_sum)
+t.test(b_pi_completers$b_fas_sum, b_pi_completers$pi_fas_sum, paired = TRUE)
+b_pi_completers %>%
+  select(b_fas_sum, pi_fas_sum) %>%
+  mutate(fas_sum_change = pi_fas_sum - b_fas_sum) %>%
+  summarize(across(everything(), list(mean = mean, sd = sd))) %>%
+  mutate(d_z = fas_sum_change_mean / fas_sum_change_sd,
+         d_av = fas_sum_change_mean / ((b_fas_sum_sd + pi_fas_sum_sd) / 2))
+
+## Body Dissatisfaction 
+plot(b_pi_completers$b_bsss_sum, b_pi_completers$pi_bsss_sum)
+t.test(b_pi_completers$b_bsss_sum, b_pi_completers$pi_bsss_sum, paired = TRUE)
+b_pi_completers %>%
+  select(b_bsss_sum, pi_bsss_sum) %>%
+  mutate(bsss_sum_change = pi_bsss_sum - b_bsss_sum) %>%
+  summarize(across(everything(), list(mean = mean, sd = sd))) %>%
+  mutate(d_z = bsss_sum_change_mean / bsss_sum_change_sd,
+         d_av = bsss_sum_change_mean / ((b_bsss_sum_sd + pi_bsss_sum_sd) / 2))
 
 
+## Demographics
 
-####  Analysis  ####
-## Functional Appreciation Scale
-plot(clean_data$b_fas_sum, clean_data$pi_fas_sum)
-t.test(clean_data$b_fas_sum, clean_data$pi_fas_sum)
+# Sex assigned at birth
+b_pi_completers %>%
+  count(b_dem_sex) %>%
+  mutate(percent = (n / sum(n))*100)
+
+# Gender 
+b_pi_completers %>%
+  select(starts_with("b_dem_gender")) %>%
+  select(-"b_dem_gender_16_TEXT") %>%
+  replace(is.na(.), 0) %>%
+  summarise(across(everything(), sum)) %>%
+  pivot_longer(everything(), names_to = "gender", values_to = "n") %>% 
+  mutate(percent_all_selections = (n / sum(n))*100) %>%  #is it correct to use the sum that accounts for people selecting multiple choices (n=146)...
+  mutate(percent_of_respondents = (n / 75)*100) #...or should we use the number of respondents (n=75). where to get this n using code?
+
+#Sexual orientation
+b_pi_completers %>%
+  count(b_dem_orientation) %>%
+  mutate(percent = (n / sum(n))*100)
+
+#Race and ethnicity
+b_pi_completers %>%
+  select(starts_with("b_dem_race_ethnicity")) %>%
+  select(-"b_dem_race_ethnicity_7_TEXT") %>%
+  replace(is.na(.), 0) %>%
+  summarise(across(everything(), sum)) %>%
+  pivot_longer(everything(), names_to = "race/ethnicity", values_to = "n") %>% 
+  mutate(percent_all_selections = (n / sum(n))*100) %>%  #is it correct to use the sum that accounts for people selecting multiple choices (n=97)...
+  mutate(percent_of_respondents = (n / 75)*100) #...or should we use the number of respondents (n=75). where to get this n  using code?
```

Comment:
Again, this latter version is correct but I would suggest using `summarize(across(everything(), mean))`

Reply:
Just curious, why is it better practice to summarize this with means instead of sums?

Reply:
If you use `summarize(across(everything(), mean))` at line 223, you don't need to have the subsequent lines at all!

Reply:
Ooooooohhhhh got it! Wonderful :)